exporter/stats/stackdriver/stackdriver.go

```diff
@@ -308,8 +308,12 @@ func newTypedValue(view *stats.View, r *stats.Row) *monitoringpb.TypedValue {
 			DoubleValue: float64(*v),
 		}}
 	case *stats.MeanData:
-		return &monitoringpb.TypedValue{Value: &monitoringpb.TypedValue_DoubleValue{
-			DoubleValue: v.Mean,
+		return &monitoringpb.TypedValue{Value: &monitoringpb.TypedValue_DistributionValue{
+			DistributionValue: &distributionpb.Distribution{
```

Comment:
You will also need to set BucketOptions and BucketCounts, similar to the DistributionData case below.

Bounds can just be {0} and BucketCounts {0, int64(v.Count)}.

Reply:
I see, updated.